src/worldstate/models/arbitration.rs

```diff
@@ -45,7 +45,17 @@ mod test {
 
         match client.fetch::<Arbitration>().await {
             Ok(_arbitration) => Ok(()),
-            Err(why) => Err(why),
+            Err(why) => {
+                if let ApiError::ApiError(error) = why {
+                    if error.code == 404 {
+                        Ok(())
+                    } else {
+                        Err(ApiError::ApiError(error))
+                    }
```

Comment:
shouldn't need this anymore cause of wfcd/warframe-worldstate-parser#531

Reply:
It still error'd with a 404 :/

Reply:
hmm, interesting. ok.